src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -117,12 +176,37 @@ def validate(self, attrs: dict[str, Any]) -> dict[str, Any]:
                     code="invalid",
                 )
 
-        if not ("medewerker_roltype" in attrs and "zaaktype" in attrs):
-            return attrs
-
         # We know it exists thanks to the previous check
         group_config = ZGWRegistration.get_zgw_config(attrs)
 
+        # Make sure the property (eigenschap) related to the zaaktype exists
+        if connections := attrs.get("property_mappings"):
```

Comment:
s/connections/mappings

Connection makes me think of HTTP/TCP connections and especially with the client interaction further this may be confusing.